### ISSUE TITLE
How to change focus outline color?

### ISSUE THREAD

**Issue by flaurida** [grommet/grommet#3079]
I would like to change the focus outline color from the default green, but cannot figure out how to do so. I tried changing the accent colors in my theme to no avail.

I would expect changing some color property would change the focus outline color, but I cannot find it. Please help? Shown below is the screenshot of the outline that I want to change the color of.

<img width="1209" alt="Screen Shot 2019-05-02 at 8 50 23 PM" src="https://user-images.githubusercontent.com/6321017/57115302-280d8f80-6d1c-11e9-8f7a-58c6d8fe2445.png">

- Grommet version: 2.0

**Comment by thomasin** [comment]
It is `global.focus.border.color`

**Comment by ShimiSun** [comment]
Check this example to get an idea of how it is done https://codesandbox.io/s/m3qp1q15z9.
Let me know if you need any further help.

**Comment by flaurida** [comment]
beautiful, thanks solved it! for my reference, is [this](https://github.com/grommet/grommet/wiki/Grommet-v2-theming-documentation) the current documentation on how to update theme?

**Comment by CharlieDeepk** [comment]
> Check this example to get an idea of how it is done https://codesandbox.io/s/m3qp1q15z9.
> Let me know if you need any further help.

hey buddy I'm too facing this problem

**Comment by dherault** [comment]
It seems that `global.focus.border.color` is not working atm.

**Comment by jcfilben** [comment]
Hi @dherault here is a [codesandbox](https://codesandbox.io/s/grommet-v2-template-forked-dkjh92?file=/index.js) using the latest version of grommet that shows `global.focus.border.color` working. Hope that helps and let us know if there are any additional questions
